.gitignore

```diff
@@ -17,4 +17,8 @@ testdbfile
 .sass-cache
 *.DS_Store
 *.jshintrc
-portal/tests/chromedriver
\ No newline at end of file
+portal/tests/chromedriver
+
+db.sqlite3
```

Comment:
These are present in the RR `.gitignore`, everyone will generate them after running locally, and they should not be committed.